src/webrtc/call.ts

```diff
@@ -325,6 +329,7 @@ export type CallEventHandlerMap = {
     /* @deprecated */
     [CallEvent.HoldUnhold]: (onHold: boolean) => void;
     [CallEvent.SendVoipEvent]: (event: VoipEvent, call: MatrixCall) => void;
+    [CallEvent.PeerConnectionCreated]: (peerConn: RTCPeerConnection, call: MatrixCall) => void;
```

Comment:
It's a pragmatic approach.  By exposing the entire object, we could have left the stats collecting to the calling client.

Reply:
Yeah, this seemed easier and most flexible. We already expose the peer connection anyway, there was just no way to know as soon as it was created.